pgoapi/location.py

```diff
@@ -27,9 +27,28 @@ def get_route(start, end, use_google=False, GMAPS_API_KEY="", walk_to_all_forts=
         else:
             d = directions_service.directions(origin, destination, mode="walking", units="metric")
         steps = d[0]['legs'][0]['steps']
-        return [(step['end_location']["lat"], step['end_location']["lng"]) for step in steps]
+        final_steps = [
```

Comment:
ok, I guess for now I'll just remove the jumping code but leave the SPIN ALL feature since it is helpful for now even if some are out of range.